Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -2291,6 +2291,13 @@ extension SourceKitLSPServer {
     }
     .sorted(by: { $0.name < $1.name })
 
+    if typeHierarchyItems.isEmpty {
+      // When returning an empty array, VS Code fails with the following two errors. Returning `nil` works around those
```

Comment:
I assume this is because it always expects at least the original element itself? Does call hierarchy have the same issue?

Reply:
Call hierarchy doesn’t seem to have that issue.